### ISSUE TITLE
SQS broker keeps creating new queues on SQS

### ISSUE THREAD

**Issue by scardena** [celery/celery#7143]
I am using SQS as a broker, and trying to use an existing queue. but everytime I am trying to send a message to the queue, celery creates a new queue with a random id as name.
This is how my my tasks.py looks like

```
from celery import Celery
from kombu.utils.url import safequote

# real credentials, faking them here.
aws_access_key = safequote("ABCDEFGHJIKL")
aws_secret_key = safequote("abcdefhjkiglkiominoql")
broker_transport_options = {'region': 'us-west-2'}

broker_url=f"sqs://{aws_access_key}:{aws_secret_key}@"
task_create_missing_queues = False
event_queue_prefix = None

app = Celery('tasks', broker=broker_url, backend='rpc://')

app.conf.broker_transport_options = broker_transport_options

app.conf.task_default_queue = 'myqueue'
app.conf.task_create_missing_queues = False
app.conf.event_queue_prefix = None

@app.task
def mytask(text):
    return f"the text intput was {text}"
```

Another one of my surprises is that the following still works. I never open on any tab a celery (server? process?) as all tutorials explain:
```
# I am not running this
celery -A tasks worker  --loglevel=INFO
```
The only thing I am doing is opening a pytho interpreter, and for some reason, running a `mytask.delay('myexample')` still works. I was expecting for it not to work
```
python
Python 3.9.9 (main, Nov 17 2021, 16:40:29)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tasks
# Why does the following works, regardless if I have a "celery process" running or not?
>>> tasks.mytask.delay('asdasd')
<AsyncResult: 311e6538-dd79-451a-b282-5be1b271d736>
```
so to summarize:
1. Why is this creating a new queue in SQS instead of just reusing `myqueue` declared in `app.conf.task_default_queue = 'myqueue'`?
2. why is this working, even if I don't have a celery process running?

**Comment by galCohen88** [comment]
@scardena This was discussed back then in this [PR](https://github.com/celery/kombu/pull/834), is it a duplicate?
I.e using `predefined_queue_urls` and `predefined_queue_urls` should do the trick

**Comment by scardena** [comment]
@galCohen88 I tried that, but that wasn't the reason. After reading the documentation I realized it was because using the rpc backend, it result in multiple creates being created in SQS, more over, rpc backend will create 1 queue per client, which was the behaviour I was seeing.
moving to redis, or just ditching the backend worked fine.
thanks.

**Comment by rawspa** [comment]
@scardena`moving to redis, or just ditching the backend worked fine.` 
did you switched to redis from sqs or redis as backend ?
